security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -276,6 +342,18 @@
         <help></help>
     </field>
     <field>
+      <label>DNS-01/Linode</label>
```

Comment:
Let's see how Ad's latest JavaScript magic works out. The prefix was added to have a destinction in the GUI between HTTP-01 and DNS-01 validation methods.